### PR TITLE
Use tokio::runtime::current_thread in CLI

### PR DESCRIPTION
**Pull request by ry**
Reduces tail latency in tests/http_bench.js by half.

### CODE REVIEW

**Comment by afinch7** [comment]
> Reduces tail latency in tests/http_bench.js by half.

Again?

**Comment by piscisaureus** [comment]
Sure enough... if it passes the tests!

File: src/cli.rs [hunk 1]
```diff
@@ -32,9 +35,11 @@ impl Cli {
     state: Arc<IsolateState>,
     permissions: DenoPermissions,
   ) -> Self {
+    let pool = Builder::new().pool_size(4).build();
```
Comment (bartlomieju) [inline src/cli.rs]: @ry this bit would most likely resolve problem described in #2124

**Comment by methodbox** [comment]
I don't know how to build this but I have a test environment with a ton of VMs I can test on if you can either spit out a binary or tell me how to process the build on Linux.

**Comment by ry** [comment]
I still want to support current_thread runtime in Deno... this patch is stale tho.

**Pull request closed without merging**